packages/fuel-indexer-plugin/src/wasm.rs

```diff
@@ -124,6 +127,27 @@ pub trait Entity<'a>: Sized + PartialEq + Eq + std::fmt::Debug {
         }
     }
 
+    /// Finds the first entity that satisfies the given constraints.
+    fn find(query: impl Into<QueryFragment<Self>>) -> Option<Self> {
+        let query: QueryFragment<Self> = query.into();
+        unsafe {
+            let buff = bincode::serialize(&query.to_string()).unwrap();
+            let mut bufflen = (buff.len() as u32).to_le_bytes();
+
+            let ptr =
+                ff_single_select(Self::TYPE_ID, buff.as_ptr(), bufflen.as_mut_ptr());
+
+            if !ptr.is_null() {
+                let len = u32::from_le_bytes(bufflen) as usize;
+                let bytes = Vec::from_raw_parts(ptr, len, len);
+                let data = deserialize(&bytes).unwrap();
+                Some(Self::from_row(data))
+            } else {
+                None
+            }
+        }
+    }
```

Comment:
Can we get rid of the `.unwrap()`s here? From what I can tell, the other functions in this module don't use them.